src/BUILD

```diff
@@ -185,6 +186,8 @@ cc_test(
     data = [
         "test/dummy/1/dummy.xml",
         "test/dummy/1/dummy.bin",
+        "test/sum/1/sum.xml",
```

Comment:
Maybe name it add2inoutsElementwise? Or something similar.

Reply:
How about `add_two_inputs`?

Reply:
Done